### ISSUE TITLE
Regex string breaks formatting for rest of a go document.

### ISSUE THREAD

**Issue by odoland** [worlpaker/go-syntax#6]
# Steps to replicate:

Copy paste this on top of a go file.
`var demo = regexp.MustCompile(`^(?i)(CS)(\d{3,4})`)`

<img width="223" alt="image" src="https://github.com/worlpaker/go-syntax/assets/20232292/4f52434e-3c8b-4f30-9128-990123aed9c4">

<img width="481" alt="image" src="https://github.com/worlpaker/go-syntax/assets/20232292/275ce122-5cf6-4cf8-8e27-4266cd7d2b0a">

On go 1.21.0
All extensions disabled.
VS code about.
```
Version: 1.86.0 (Universal)
Commit: 05047486b6df5eb8d44b2ecd70ea3bdf775fd937
Date: 2024-01-31T10:29:15.765Z
Electron: 27.2.3
ElectronBuildId: 26495564
Chromium: 118.0.5993.159
Node.js: 18.17.1
V8: 11.8.172.18-electron.0
OS: Darwin x64 22.6.0
```

**Comment by worlpaker** [comment]
Thanks for the details! VS Code using the old version of Go Syntax. I think this issue was fixed in v0.5.7. 
Can you please install the extension and try it? (Install from here: https://marketplace.visualstudio.com/items?itemName=furkanozalp.go-syntax)

**Dark+ Theme v0.5.7**
![test](https://github.com/worlpaker/go-syntax/assets/34960716/4db82a0f-d356-4ae7-a4be-b67f306ebd2f)

**Comment by alexr00** [comment]
It looks like VS Code insiders also already has this fix!

**Comment by alexr00** [comment]
I've also just pulled updates to the grammar into VS Code again so we have the latest in insiders.

**Comment by worlpaker** [comment]
FYI: There is a new update (v0.5.8).

**Comment by odoland** [comment]
> Thanks for the details! VS Code using the old version of Go Syntax. I think this issue was fixed in v0.5.7. Can you please install the extension and try it? (Install from here: https://marketplace.visualstudio.com/items?itemName=furkanozalp.go-syntax)
> 
> **Dark+ Theme v0.5.7** ![test](https://private-user-images.githubusercontent.com/34960716/302756952-4db82a0f-d356-4ae7-a4be-b67f306ebd2f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDczMzI4MzEsIm5iZiI6MTcwNzMzMjUzMSwicGF0aCI6Ii8zNDk2MDcxNi8zMDI3NTY5NTItNGRiODJhMGYtZDM1Ni00YWU3LWE0YmUtYjY3ZjMwNmViZDJmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjA3VDE5MDIxMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTEzMzg5MDEyODlhOGIyNjcwYTVkNzk2ZTU0YjlmMjY4ZDk0MTU1ZWQyNWYzZGU3NmU0NDc3OTRkOTk5ZGZkMWMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.xcmY-BUV0r6u_76Rva9pAyp4IeqNImu1WTJj6o8_aGY)

Thanks, grabbing this extension and installing it worked!

I'll play around in the latest insiders edition for VSCode to see if we can catch any other issues.
